src/pages/membership/components/createYouthProfileForm/CreateYouthProfileForm.tsx

```diff
@@ -249,71 +249,71 @@ function CreateYouthProfileForm(props: Props) {
                 invalidText={
                   props.submitCount && props.errors.city && t(props.errors.city)
                 }
-                labelText={t('registration.city')}
+                labelText={t('registration.city') + ' *'}
               />
             </div>
-            <div className={styles.resRow}>
-              <Field
-                className={styles.childBirthDayInput}
-                as={TextInput}
-                id="birthDay"
-                name="birthDay"
-                type="number"
-                invalid={props.submitCount && props.errors.birthDay}
-                invalidText={
-                  props.submitCount &&
-                  props.errors.birthDay &&
-                  t(props.errors.birthDay)
-                }
-                labelText={t('registration.childBirthDay')}
-              />
-              <span className={styles.birthdayMiddleDot}>&#8901;</span>
-              <Field
-                className={styles.childBirthInput}
-                as={TextInput}
-                id="birthMonth"
-                name="birthMonth"
-                hideLabel={true}
-                type="number"
-                invalid={props.submitCount && props.errors.birthMonth}
-                invalidText={
-                  props.submitCount &&
-                  props.errors.birthMonth &&
-                  t(props.errors.birthMonth)
-                }
-                labelText={t('registration.childBirthMonth')}
-              />
-              <span className={styles.birthdayMiddleDot}>&#8901;</span>
-              <Field
-                className={styles.childBirthInput}
-                as={TextInput}
-                id="birthYear"
-                name="birthYear"
-                hideLabel={true}
-                type="number"
-                invalid={props.submitCount && props.errors.birthYear}
-                invalidText={
-                  props.submitCount &&
-                  props.errors.birthYear &&
-                  t(props.errors.birthYear)
-                }
-                labelText={t('registration.childBirthYear')}
-              />
+            <div className={styles.birthDayContainer}>
+              <h6>{t('registration.birthdayHelper')}</h6>
```

Comment:
h6 feels quite unnecessary, could this be a bit more important (e.g. h3) styled differently, or just a strong?